partiql-parser/src/partiql.pest

```diff
@@ -1,5 +1,9 @@
 WHITESPACE = _{ " " | "\t" | "\x0B" | "\x0C" | "\r" | "\n" }
 
+BlockComment = { "/*" ~ ( BlockComment | (!"*/" ~ ANY) )* ~ "*/" }
+UntilEolComment = { "--" ~ ( !"\n" ~ ANY )* ~ "\n" }
+COMMENT = _{ UntilEolComment | BlockComment }
```

Comment:
Minor style suggestion edit, the rules in this file typically go from top to bottom, so this just re-ordered them to be consistent, and renames `UntilEolComment` to `LineComment`.

```suggestion
COMMENT = _{ LineComment | BlockComment }

BlockComment = { "/*" ~ ( BlockComment | (!"*/" ~ ANY) )* ~ "*/" }
LineComment = { "--" ~ ( !"\n" ~ ANY )* ~ "\n" }
```